.github/workflows/tests.yml

```diff
@@ -144,12 +144,21 @@ jobs:
     # TODO: when actions supports using ${{env}} in job.*.if and not just job.*.steps.*.if, use this:
     #if: ${{ env.NIGHTLY }}
     # in the meantime, copy-paste this:
-    if: ${{ github.event_name == 'schedule' || github.event_name == 'push' || github.event_name == 'workflow_dispatch' }}
+    # if: ${{ github.event_name == 'schedule' || github.event_name == 'push' || github.event_name == 'workflow_dispatch' }}
```

Comment:
TODO: [fa81a72](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4106/commits/fa81a72774fa259a774df740bece6aa168293a81) will need to be reverted before this PR is merged.

(Debian 9 tests typically only run on `master`; they have been temporarily enabled only to test this PR.)